debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/RecordMakers.java

```diff
@@ -173,7 +173,7 @@ public boolean assign(long tableNumber, TableId id) {
         TableSchema tableSchema = schema.schemaFor(id);
         if (tableSchema == null) return false;
 
-        String topicName = topicSelector.getTopic(id);
+        String topicName = tableSchema.getParentSchemaName();
```

Comment:
I don't fully understand what this `topicName` is used for. What is the envelope schema used for?